CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+## Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#pre-requirements)
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+- [Testing](#testing)
+- [Code of Conduct](#code-of-conduct)
+- [Getting Help](#getting-help)
+
+
+### Pre Requirements
+
+---
+
+To contribute to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations type 'vagrant up' and automatically the development environment will set up.
+
+
+### Issues
+
+---
+
+Issues should be used to report problems with the library, request a new feature, or to discuss potential changes before a PR is created.
+Please make sure that before the PR is created, the issue must be created and after the PR is created, the PR must be linked to the issue relevant to it.
+
+
+### Pull Requests
+
+---
+
+To create a pull request (PR) please follow the instructions:
+
+1. Create an [issue](#Issues).
+2. Fork a repository and clone the repository.
+3. Create a new branch and new remote for the upstream repository.
+4. Make the changes on our local machine, add to the branch, commit the a massage and finally push the PR.
+5. Open a PR in our repository and follow this process: 
+	- Add a detailed description.
+	- Press on the Pull Requests.
+6. Link the issue to the PR when you open issues use there [words](#https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
+  Issues can also help the team know what every member of the team work on.
+
+
+### Pull Requests Standards
+
+---
+Our standards for pull request is:
+
+1. Make polite notes to each other without using bad language.
```

Comment:
Can maybe rephrased to be more accurate and show a positive idea (instead of negative idea - `bad language`)